.coveragerc

```diff
@@ -22,7 +22,7 @@ omit =
   google/cloud/__init__.py
 
 [report]
-fail_under = 89
+fail_under = 95
```

Comment:
Still not 100, but it's solid progress. I don't want this PR to get too long. Mostly I started working on unit tests because they're just mindless enough that I can handle it while I'm sick this week.

Reply:
It's fine to merge a meaningful piece of work into `main`, it doesn't have to be all-or-nothing. This PR by itself is already a significant improvement.